.repo-meta.yml

```diff
@@ -16,18 +16,15 @@ software:
       - NuxtJS
```

Comment:
I'm actually curious if we need the repo-meta file anymore if our GitHub Action pipeline will create and finish RFCs for us. I believe the repo-meta process that used to populate fields into the cmdb has been turned off.

Reply:
True. I don't think we do anymore.

Reply:
I talked with Glen and we definitley don't.